policy/release/trusted_task/trusted_task.rego

```diff
@@ -307,6 +308,23 @@ _trusted_build_digests contains digest if {
 	some digest in _digests_from_values(lib.result_values(result))
 }
 
+# If an image is part of the snapshot we assume that was built in Konflux and
+# therefore it is considered trustworthy. IIUC the use case is something to do
+# with building an image in one component, and being able to use it while
+# building another component in the same application.
+_trusted_build_digests contains digest if {
+	some component in input.snapshot.components
```

Comment:
How does this `input` variable look? trying to understand its content.

Reply:
Input is the big global input with everything in it. You can see what it looks like using `--output policy-input=input.json` with ec validate image (or hack/simple-demo.sh). See also <https://conforma.dev/docs/ec-cli/policy_input.html> .